Mark9Mbugua/Politico-V
# #164030401 Test edge cases of political parties and offices

#### What does this PR do?
It adds the edge cases for the political parties and views

#### Description of Task to be completed?
Write unit tests using irregular and invalid test cases to stretch the code and observe if it is breakable.
 
#### How should this be manually tested?
- Git clone this repo in a local repo
- In the root directory, install pytest
- Observe how tests run

#### What are the relevant pivotal tracker stories?
#164030401
